Common/Texture.cs

```diff
@@ -11,9 +9,12 @@ namespace LearnOpenTK.Common
     public class Texture
     {
         public readonly int Handle;
+        public string type;
+        public string path;
 
-        public static Texture LoadFromFile(string path)
+        public static Texture LoadFromFile(string filename, string type = "texture_diffuse")
```

Comment:
We generally avoid default arguments in opentk code. Also I don't think this field is a good idea, and if we where going to do it I would rather this be an enum rather than a string.